CHANGES.md

```diff
@@ -1,6 +1,8 @@
 # Unreleased
 - [NEW] Constructor for `Database` subclasses.
 - [IMPROVED] Documentation for `Database.findByIndex` to show complete selector.
+- [FIX] `CouchDbException: 400 Bad Request: bad_request: invalid_json` when a
+  query parameter contains a semicolon.
```

Comment:
It was just `;` the `=`,`+`,`&` were already there, but the diff made it show up badly and it was undocumented as to why they were special.